bin/helpers

```diff
@@ -61,6 +61,23 @@ waitInstanceBooted() (
   return 1 # Failed.
 )
 
+# systemdClean: wait for instance to be booted and check for systemd failures.
+# ==> Returns true if systemd state is clean (no failure), false otherwise.
+systemdClean() (
```

Comment:
Just a style nit really, but I tend to add "is" as a prefix for functions answering boolean questions like this, as otherwise it can look like this function is "cleaning systemd" (what ever that means) when reading its usage in situ.